core/modules/replica/DatabaseServicesMySQL.h

```diff
@@ -348,6 +366,12 @@ class DatabaseServicesMySQL : public DatabaseServices {
     std::vector<TransactionInfo> _findTransactionsImpl(util::Lock const& lock,
                                                        std::string const& predicate);
 
+    TransactionContribInfo _transactionContribImpl(util::Lock const& lock,
+                                                   std::string const& predicate);
+
+    std::vector<TransactionContribInfo> _transactionContribsImpl(util::Lock const& lock,
+                                                                 std::string const& predicate);
```

Comment:
are these Impl functions still needed? I can't tell that they're being called anywhere.

Reply:
The implementation is in the corresponding `CC` file `core/modules/replica/DatabaseServicesMySQL.cc`:
```
vector<TransactionContribInfo> DatabaseServicesMySQL::_transactionContribsImpl(
        util::Lock const& lock,
        string const& predicate) {

    string const context = _context(__func__) + "predicate=" + predicate + " ";
    ...
```

Reply:
huh, I thought I looked there. Ok, thx.